src/SanitizerSvg.php

```diff
@@ -22,41 +22,41 @@ class SanitizerSvg {
 			'width', 'height', 'viewBox', 'xmlns', 'version', 'preserveAspectRatio'
 		],
 		'rect' => [
-			'x', 'y', 'width', 'height', 'rx', 'ry', 'fill', 'stroke', 'stroke-width'
+			'x', 'y', 'width', 'height', 'rx', 'ry', 'fill', 'stroke', 'stroke-width', 'style'
```

Comment:
the SVG from example has color defined like that:

```
<polygon style="fill:#F95428;" points="132.7...
```
so we need to enable `style` attr for kses. and then enable it for filter `safe_style_css` which removes all non-standard CSS properties